.github/workflows/ci.yaml

```diff
@@ -463,13 +463,85 @@ jobs:
           VERSION=$(./hack/get_version_from_git.sh)
           # "+" character is not allowed in OBS dockerfile version strings
           VERSION=${VERSION//[+]/-}
-          sed -i 's~%%VERSION%%~'"${VERSION}"'~' packaging/suse/Dockerfile
+          sed -i 's~%%VERSION%%~'"${VERSION}"'~' $FOLDER/Dockerfile
       - name: Commit on OBS
         run: |
           OBS_PACKAGE=$OBS_PROJECT/$NAME
           osc checkout $OBS_PACKAGE -o $DEST_FOLDER
-          cp -r packaging/suse/* $DEST_FOLDER
+          cp -r $FOLDER/* $DEST_FOLDER
           tar --transform 's,^./,/wanda/,' -zcvf $DEST_FOLDER/wanda.tar.gz --exclude=./.git ./*
           cd $DEST_FOLDER
           osc ar
           osc commit -m "New development version of $NAME released"
+
+  obs-commit-rpm:
+    name: Commit the RPM package on OBS
+    needs: [static-code-analysis, test]
+    runs-on: ubuntu-20.04
+    if: github.event_name == 'release' || (github.event_name == 'push' && github.ref_name == 'main') || github.event_name == 'workflow_dispatch'
+    container:
+      image: ghcr.io/trento-project/continuous-delivery:main
+      env:
+        OBS_USER: ${{ secrets.OBS_USER }}
+        OBS_PASS: ${{ secrets.OBS_PASS }}
+        OBS_PROJECT: ${{ secrets.OBS_PROJECT }}
+        OSC_CHECKOUT_DIR: /tmp/trento-wanda-package
+        REPOSITORY: ${{ github.repository }}
+        FOLDER: packaging/suse/rpm
+      options: -u 0:0
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.12.1
+        with:
+          access_token: ${{ github.token }}
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Configure OSC
+        run: |
+          mkdir -p $HOME/.config/osc
+          cp /home/osc/.config/osc/oscrc $HOME/.config/osc
+          /scripts/init_osc_creds.sh
+      - name: Prepare _service file
+        run: |
+          git config --global --add safe.directory /__w/wanda/wanda
+          VERSION=$(./hack/get_version_from_git.sh)
+          sed -i 's~%%REVISION%%~${{ github.sha }}~' $FOLDER/_service && \
+          sed -i 's~%%REPOSITORY%%~'"${REPOSITORY}"'~' $FOLDER/_service && \
+          sed -i 's~%%VERSION%%~'"${VERSION}"'~' $FOLDER/_service
+      - name: Setup rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: 1.66
+      - name: Get mix deps
+        run: mix local.hex --force && mix deps.clean --all && mix deps.get
```

Comment:
Should we install `rebar` as we do in the docker style?
Or is it not needed?

Reply:
I think it's only used in the build phase. We do install it as a BuildRequire in OBS (see the spec file) and it seems to be enough based on the fact that the resulting binary seems to work correctly